.github/workflows/validation.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         run: export PIP_USER=0; SKIP=black,isort,flake8,pyright pre-commit run --all-files
 
       - name: Run black formatter check
-        run: black --check .
+        run: black --check --diff .
```

Comment:
`--check` doesn't make effect anymore.

Reply:
It does, `--diff` doesn't end with error code 1 on failures, which means the status check would see black passing, even though the diff would be shown